{
  "contest": 125,
  "handle": "hyh",
  "address": "0x859e1Dfb430A7156fAEF11947F2FC2a3C34B733A",
  "risk": "3",
  "title": "LidoVault's _withdrawFromYieldPool doesn't check ETH transfer call success",
  "issueId": 74,
  "issueUrl": "https://github.com/code-423n4/2022-05-sturdy-findings/issues/74"
}